test/solveEquation/solveEquation.test.js

```diff
@@ -58,6 +58,12 @@ describe('solveEquation for =', function () {
     ['2(x+3)/3 = 2', 'x = 0'],
     ['( u )/( 0.3) = 4u + 6.28', 'u = -9.42'],
     ['- q - 4.36= ( 2.2q )/( 1.8)', 'q = -1.962'],
+    //['2/x = 1', 'x = 2'], TODO: fixme
+    //['2/(4x) = 1', 'x = 1/2'], TODO: bombs out
+    //['2/(8 - 4x) = 1/2', 'x = 1'], TODO: fixme
+    //['2/(1 + 1 + 4x) = 1/3', 'x = 1'], TODO: fixme
+    //['(3 + x) / (x^2 + 3) = 1', '-x^2 + x = 0'], TODO: bombs out
```

Comment:
eventually it'd be nice to not split the numerator like that, but I'm not sure when we'd want and when we wouldn't